Add Webb.ai integration tile (#2407)
DataDog/integrations-extras: webb_ai/README.md

# Webb.ai

## Overview

> Matt is the first AI-enabled reliability engineer that automates troubleshooting to identify the root cause in <5 mins. Matt troubleshoots alerts from Observability tools (e.g., Datadog), cloud providers (e.g., AWS), and infrastructure (e.g., K8s).

Reviewer: ```suggestion
Matt by Webb.ai is the first AI-enabled reliability engineer. It automates troubleshooting to identify the root cause in less than 5 mins. Matt troubleshoots alerts from Observability tools like Datadog, cloud providers like AWS, Azure, and Google Cloud, and infrastructure like Kubernetes.
```

Also, the root cause of what? An incident? Please add that.

## Setup

> Visit [Webb.ai][2] to sign up for free. Once registered, you can start the integration from Webb.ai tile in Datadog integrations page.

Reviewer: ```suggestion
Visit [Webb.ai][2] to sign up for free. Once you register, you can start the integration from the Webb.ai tile on the Datadog integrations page.
```

 

## Uninstallation

> To remove the Datadog integration from Webb.ai, navigate to the [Webb.ai integrations page][1] and delete Datadog integration from the list.

Reviewer: ```suggestion
To remove the Datadog integration from Webb.ai, navigate to the [Webb.ai integrations page][1] and delete the Datadog integration from the list.
```

> - Once this integration has been uninstalled, any previous authorizations are revoked. 
> - Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the Datadog API Keys page.

Reviewer: ```suggestion
Once this integration has been uninstalled, any previous authorizations are revoked. 

To ensure that all API keys associated with this integration are disabled, search for the integration name on the [Datadog API Keys][4] page.
```

## Data Collected

### Metrics

> Webb.ai does not generate any metrics and therefore relies on Datadog metrics.

Reviewer: ```suggestion
Webb.ai does not generate any metrics; it relies on Datadog metrics.
```

### Service Checks
Webb.ai does not include any service checks.

### Events
Webb.ai does not generate any events.

## Troubleshooting

Need help? Contact [Webb.ai support][3].

[1]: https://app.webb.ai/integrations
[2]: https://app.webb.ai/
[3]: mailto:support@webb.ai

> 

Reviewer: ```suggestion
[4]: https://app.datadoghq.com/organization-settings/api-keys

```

# Webb.ai

## Overview

> Matt by Webb.ai is the first AI-enabled reliability engineer. Matt troubleshoots alerts from observability tools like Datadog, cloud providers like AWS, Azure, and Google Cloud, and incidents from infrastructure like Kubernetes. It automates troubleshooting to identify the root cause of an alert/incident in less than 5 mins.

Reviewer: ```suggestion
Matt by Webb.ai is the first AI-enabled reliability engineer. Matt troubleshoots alerts from observability tools like Datadog, cloud providers like AWS, Azure, and Google Cloud, and incidents from infrastructure like Kubernetes. It automates troubleshooting to identify the root cause of an alert or incident in less than 5 mins.
```

> Matt by Webb.ai is the first AI-enabled reliability engineer. Matt troubleshoots alerts from observability tools like Datadog, incidents from infrastructure like Kubernetes, and alerts from cloud providers like AWS, Azure, and Google Cloud. It automates troubleshooting to identify the root cause of an alert or incident in less than 5 mins.

Reviewer 2: The overview should include: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What specific data your integration monitors and the value?

## Setup

Visit [Webb.ai][2] to sign up for free. Once you register, you can start the integration from the Webb.ai tile on the Datadog integrations page.

>  

Reviewer 2: This section should list out the exact steps the user has to take to install the integration. For example, 1. Click Install on the tile, 2. Click connect accounts 3. (oauth steps).... etc

## Uninstallation
To remove the Datadog integration from Webb.ai, navigate to the [Webb.ai integrations page][1] and delete the Datadog integration from the list.

Once this integration has been uninstalled, any previous authorizations are revoked.

To ensure that all API keys associated with this integration are disabled, search for the integration name on the [Datadog API Keys][4] page.

## Data Collected

### Metrics
Webb.ai does not generate any metrics; it relies on Datadog metrics.

### Service Checks
Webb.ai does not include any service checks.

### Events

> Webb.ai does not generate any events.

Reviewer 2: Please update this to reflect what data you are sending in

# Webb.ai

## Overview

Matt by Webb.ai is the first AI-enabled reliability engineer. Matt troubleshoots Monitors from Datadog, incidents from infrastructure like Kubernetes, and alerts from cloud providers like AWS, Azure, and Google Cloud. It automates troubleshooting to identify the root cause of an alert or incident in less than 5 mins.

With this integration
- Datadog monitors in alert state will be identified and automatically analyzed by Matt
- Troubleshooting results will improve as Matt can query Datadog metrics when required(even for kubernetes incidents, aws alerts etc.)

Data queried by Matt:

> - Event stream to identify Monitors in alert state

Reviewer 2: Please include what data is specifically sent to datadog/what data is being monitored in Datadog and the value

# Webb.ai

## Overview

Matt by Webb.ai is the first AI-enabled reliability engineer.
Matt troubleshoots monitors and alerts from Datadog, Kubernetes, and cloud providers like AWS, Azure, and GCP.
It identifies the root cause of an alert or incident in less than 5 mins.

With this integration, Matt automatically identifies alerts from Datadog and troubleshoots them. The mean time to debug of those alerts will be reduced significantly.

> Data queried by this integration:

Reviewer: ```suggestion
This integration queries the following data:
```

> With this integration, Matt automatically identifies alerts from Datadog and troubleshoots them, and the mean time to debug those alerts is reduced significantly.
> 
> This integration queries the following data:
> - Datadog events
> - Datadog metrics and tags

Reviewer 2: ```suggestion
This integration pulls data from Datadog include events, metrics, and tags. Matt utilizes the queried data to troubleshoot and debug alerts decreasing MTTR. 
```

> This integration sends events to Datadog include the root cause analysis performed by Matt and changes in the Kubernetes clusters.

Reviewer: ```suggestion
This integration sends events to Datadog, including the root cause analysis performed by Matt and changes in Kubernetes clusters.
```

> Customers are able to view detailed RCA analysis for their alerts, including all hypotheses analyzed and the exact steps Matt took with supporting evidence.

Reviewer: ```suggestion
You can view detailed RCA analysis for your alerts, including all hypotheses analyzed and the exact steps Matt took with supporting evidence.
```

> This integration sends events to Datadog, including the root cause analysis performed by Matt and changes in Kubernetes clusters.
> You can view detailed RCA analysis for your alerts, including all hypotheses analyzed and the exact steps Matt took with supporting evidence.
> With this analysis, 80-90% of troubleshooting is automated, greatly reducing the debugging time of on-call engineers. 

Reviewer 2: ```suggestion
With this integration, events are sent to Datadog including the root cause analysis performed by Matt and changes in Kubernetes clusters. The events include detailed RCA analysis for your alerts such as the steps Matt took and all analyzed hypotheses to reduce debugging time of on-call engineers. 
```

Reviewer: This edit changes the meaning of the prose and I don't think it's correct.

> With this analysis, 80% - 90% of the troubleshooting are automated, thus greatly reducing the debugging time of oncall engineers. 

Reviewer: ```suggestion
With this analysis, 80-90% of troubleshooting is automated, greatly reducing the debugging time of on-call engineers. 
```

## Setup

> 1. Visit [Webb.ai][2] to sign up for free

Reviewer: ```suggestion
1. Visit [Webb.ai][2] and sign up for the free service.
```

> 2. Navigate to Webb.ai tile in [Datadog integrations page][5] and click **Install Integration**

Reviewer: ```suggestion
2. Navigate to Webb.ai tile in the [Datadog integrations page][5] and click **Install Integration**.
```

> 3. Go to the **Configure** tab and click **Connect Accounts**

Reviewer: ```suggestion
3. Go to the **Configure** tab and click **Connect Accounts**.
```

> 4. Follow the series of OAuth steps to complete the integration

Reviewer: ```suggestion
4. Follow the series of OAuth steps to finish setting up the integration.
```

## Uninstallation
To remove the Datadog integration from Webb.ai, navigate to the [Webb.ai integrations page][1] and delete the Datadog integration from the list.

Once this integration has been uninstalled, any previous authorizations are revoked.

To ensure that all API keys associated with this integration are disabled, search for the integration name on the [Datadog API Keys][4] page.

## Data Collected

### Metrics
Webb.ai does not generate any metrics; it relies on Datadog metrics.

### Service Checks
Webb.ai does not include any service checks.

### Events
Webb.ai sends the following events to Datadog:

> - RCAs performed by Matt

Reviewer: ```suggestion
- Root cause analyses performed by Matt
```

# Webb.ai

## Overview

Matt by Webb.ai is the first AI-enabled reliability engineer.
Matt troubleshoots Monitors from Datadog, incidents from infrastructure like Kubernetes, 
and alerts from cloud providers like AWS, Azure, and Google Cloud.
It automates troubleshooting to identify the root cause of an alert or incident in less than 5 mins.

With this integration, MTTR for Datadog customers will be reduced as Matt identifies Datadog alerts and automatically troubleshoots them, typically in less than 5min

Data queried by Matt:
- Events to identify Monitors in alert state
- Monitors, Metrics and their values

Matt sends the following events to Datadog:
- RCAs performed by Matt

> - Changes observed in Kubernetes clusters

Reviewer 2: This is a great start but it would be helpful to provide the value in 1-2 sentences of the data being monitored. EX: 

This integration sends events to Datadog including RCAs performed by Matt and changes in kubernetes clusters <insert value of this information here> 

Ex of value: providing insight into xyz

Reviewer 2: We want to inform the user as to why this information is helpful to them

Reviewer 3: @emarsha94 Thanks for the feedback. We updated the text to reflect why we think this is useful to datadog users. Please take a look.
